### ISSUE TITLE
Migrate the scripts in fabric8-launcher/launcher-openshift-templates to launcher-backend repository

### ISSUE THREAD

**Issue by gastaldi** [fabric8-launcher/launcher-application#828]
Now that the launcher-frontend and the creator-backend are part of the launcher-backend service and as part of our service unification efforts, it would be nice to move the contents of the https://github.com/fabric8-launcher/launcher-openshift-templates repository under a subdirectory inside the launcher-backend repository. The scripts need to be changed also to reflect this single service approach.

@quintesse I am assigning this issue to you since you're the original author. Is this something you can have a look at?

**Comment by quintesse** [comment]
Of course, I was already on it 😊

**Comment by quintesse** [comment]
So, I did a first version of this that at least seems to create a working Launcher if I use `./scripts/minishift.install.sh --latest`.

But I still see the same problem I mentioned before:

 - if I create an app (frontend Vue, backend Quarkus with all capabilities selected) I get en error message in the frontend. The deploy did seem to work though (both to my minishift and to GitHub)
 - Nothing in the Welcome App seems to work (URLs, buttons, etc)

And another one I noticed:

 - The database didn't finish starting

So it would be nice if you guys (@gastaldi @edewit @ia3andy) tested this as well to see if you're getting the same (or other) problems.
